### ISSUE TITLE
Split css in multiple files doesn't work with webpacker

### ISSUE THREAD

**Issue by renanbronchart** [rails/webpacker#1876]
I want to split my application.css just after compile sass in css.
I had CSSSplitWebpackPlugin in webpack config

### my webpack custom.js
```
const path = require('path')
const webpack = require('webpack')
const CSSSplitWebpackPlugin = require('css-split-webpack-plugin').default

module.exports = {
  plugins: [
    new CSSSplitWebpackPlugin()
  ]
}

```

### my webpacker.yml

```
# Note: You must restart bin/webpack-dev-server for changes to take effect

default: &default
  source_path: app/javascript
  source_entry_path: packs
  public_output_path: packs
  cache_path: tmp/cache/webpacker

  # Additional paths webpack should lookup modules
  # ['app/assets', 'engine/foo/app/assets']
  resolved_paths: []

  # Reload manifest.json on all requests so we reload latest compiled packs
  cache_manifest: false

  extensions:
    - .vue
    - .js
    - .sass
    - .scss
    - .css
    - .module.sass
    - .module.scss
    - .module.css
    - .png
    - .svg
    - .gif
    - .jpeg
    - .jpg

development:
  <<: *default
  compile: true

  # Reference: https://webpack.js.org/configuration/dev-server/
  dev_server:
    https: false
    host: localhost
    port: 3035
    public: localhost:3035
    hmr: false
    # Inline should be set to true if using HMR
    inline: true
    overlay: true
    compress: true
    disable_host_check: true
    use_local_ip: false
    quiet: false
    headers:
      'Access-Control-Allow-Origin': '*'
    watch_options:
      ignored: /node_modules/


test:
  <<: *default
  compile: true

  # Compile test packs to a separate directory
  public_output_path: packs-test

production:
  <<: *default

  # Production depends on precompilation of packs prior to booting for performance.
  compile: false

  # Cache manifest.json for performance
  cache_manifest: true

staging:
  <<: *default

  compile: false
  cache_manifest: true
 
```

### In application.html.slim
```
= stylesheet_pack_tag 'application', media: 'all'
```

But it seems not work. 
Manifest just add application.css but not application-1, application-2, application-3, etc ...

Thanks a lot for reply

**Comment by jakeNiemiec** [comment]
It looks like there are others having problems using `CSSSplitWebpackPlugin ` with Rails: https://github.com/metalabdesign/css-split-webpack-plugin/issues/27

You might try experimenting with the new `cacheGroups`, the successor to [the removed CommonsChunkPlugin](https://medium.com/webpack/webpack-4-released-today-6cdb994702d4). Here are the docs for `cacheGroups`: https://webpack.js.org/plugins/split-chunks-plugin/#splitchunks-cachegroups

**Comment by liang3404814** [comment]
@renanbronchart I'm not so super familiar with `CSSSplitWebpackPlugin`, but I do use `environment.splitChunks();` and notice the same issue. After some investigation I was able to fix it using `javascript_packs_with_chunks_tag` (https://github.com/rails/webpacker/blob/master/README.md) in place of of `stylesheet_pack_tag`. Hope this helps.
